mne/dipole.py

```diff
@@ -386,11 +400,18 @@ def constraint(rd):
     amp = np.sqrt(np.sum(Q * Q))
     norm = 1 if amp == 0 else amp
     ori = Q / norm
+
+    dist = _compute_nearest(surf['rr'], rd_final[np.newaxis, :],
+                            return_dists=True)[1][0]
+    logger.info('---- Fitted : %7.1f ms, distance to inner skull : %2.4f mm' %
+                (1000 * t, dist * 1000))
     return rd_final, amp, ori, gof, residual
 
 
 @verbose
-def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
+def fit_dipole(evoked, cov, bem, trans=None, min_dist=5,
```

Comment:
a min_dist='auto' would be nice indeed. For this PR or a next one.

for now the urgent need is a test